docs/pull_request.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request
+
+To contribute for CloudCV in EvalAI is a facile task. Please follow these steps in order to get started :-)
```

Comment:
Please replace with `Contributing to EvalAI is a facile task. Please follow these steps in order to get started.`